Resources/Prototypes/_Sunrise/roadmap.yml

```
@@ -1,192 +1,117 @@
-          state: Planned
-        ussp:
-          id: ussp
-          name: USSP Station
-          desc: A separate station of the USSP with its own shift tasks.
-          state: Planned
-        ratvar:
-          id: ratvar
-          name: Ratvar Cult
-          desc: A cult worshipping a mechanical god.
-          state: Planned
+  - name: roadmap-available-now
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)